sentry/src/main/java/io/sentry/SentryReplayOptions.java

```diff
@@ -49,30 +52,28 @@ public enum SentryReplayQuality {
   private @Nullable Double onErrorSampleRate;
 
   /**
-   * Redact all text content. Draws a rectangle of text bounds with text color on top. By default
-   * only views extending TextView are redacted.
+   * Redact all views with the specified class names. The class name is the fully qualified class
+   * name of the view, e.g. android.widget.TextView. The subclasses of the specified classes will be
+   * redacted as well.
    *
-   * <p>Default is enabled.
-   */
-  private boolean redactAllText = true;
-
-  /**
-   * Redact all image content. Draws a rectangle of image bounds with image's dominant color on top.
-   * By default only views extending ImageView with BitmapDrawable or custom Drawable type are
-   * redacted. ColorDrawable, InsetDrawable, VectorDrawable are all considered non-PII, as they come
-   * from the apk.
+   * <p>If you're using an obfuscation tool, make sure to add the respective proguard rules to keep
+   * the class names.
    *
-   * <p>Default is enabled.
+   * <p>Default is empty.
    */
-  private boolean redactAllImages = true;
+  private Set<String> redactClasses = new CopyOnWriteArraySet<>();
```

Comment:
@brustolin @bruno-garcia I guess we should align on naming here. Do you prefer this or `redactViewTypes`? I'm fine with either

Reply:
I think we should align but just Classes is too generic, what about "redactViewClasses"?

Reply:
yeah, sounds good. I can also change it to `redactViewTypes` so you don't have to :)